apps/i18n/common/en_us.json

```diff
@@ -2653,6 +2653,7 @@
   "startOver": "Start Over",
   "startOverTitle": "Are you sure you want to start over?",
   "startOverBody": "This will reset the puzzle to its start state and reset all the data you've added or changed.",
+  "startOverAichatModelCustomizations": "This will reset this level to its start state and remove any model customizations or model card information you’ve added or changed.",
```

Comment:
Minor comment for a potential follow-up - might be better to have this string in the aichat specific translation file since this is a common file?